crates/bevy_render/src/renderer/mod.rs

```diff
@@ -112,6 +112,13 @@ pub struct RenderTextureFormat(pub wgpu::TextureFormat);
 #[derive(Resource, Clone, Deref, DerefMut)]
 pub struct AvailableTextureFormats(pub Arc<Vec<wgpu::TextureFormat>>);
 
+#[cfg(target_os = "linux")]
+const GPU_NOT_FOUND_ERROR_MESSAGE: &str = "Unable to find a GPU! Make sure you have installed required drivers! For extra information, see: https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md";
```

Comment:
Should this link to main or to latest?

Reply:
Linking to `latest` is probably better, since it's what most of the users are going to use and see.

Though if someone is compiling bevy from `main`, and we add a new dependency, `latest` might not reflect that and make them confused?